azure/durable_functions/tasks/task_utilities.py

```diff
@@ -2,6 +2,7 @@
 from ..models.history import HistoryEventType
 from ..constants import DATETIME_STRING_FORMAT
 from azure.functions._durable_functions import _deserialize_custom_object
+from datetime import datetime
```

Comment:
So this is the new datetime format we are using. Is there anyway for us to remove the reference to the old datetime format so it isn't accidentally used?

Reply:
This is actually the same datetime format as before, I just needed to explicitly import it now, because I needed to use its `datettime.strptime` parser. 

I'll write some more thoughts on the "two datetime variants" in a comment below.